portfolio/src/main/webapp/script.js

```diff
@@ -132,3 +141,27 @@ function addLogInOutButton(url, isLoggedIn) {
   }
   
 }
+
+function addCustomizations(container) {
+  const customElementsHTML  = 
```

Comment:
This inline html in javascript does not spark joy. Maybe you could inline this in your html page and toggle the visibility with style display? like this? https://www.w3schools.com/howto/howto_js_toggle_hide_show.asp

Or you could have this in a separate static html page and fetch it from your server (not with a servlet, just static content)